plenum/server/consensus/ordering_service.py

```diff
@@ -2310,6 +2310,26 @@ def process_view_change_started(self, msg: ViewChangeStarted):
 
     def process_new_view_accepted(self, msg: NewViewAccepted):
         self._write_manager.future_primary_handler.set_primaries(msg.view_no, self._data.primaries)
+        self._setup_for_non_master_after_view_change(msg.view_no)
+
+    def _setup_for_non_master_after_view_change(self, current_view):
```

Comment:
Should we just delete all `stashed_out_of_order_commits` before we start a view change? In `_clear_all_3pc_msgs` for example.